pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -2,15 +2,14 @@ package knativeserving
 
 import (
 	"context"
-	"errors"
 	"flag"
 	"fmt"
 	"time"
 
 	mf "github.com/jcrossley3/manifestival"
-	maistrav1 "github.com/maistra/istio-operator/pkg/apis/maistra/v1"
 	servingv1alpha1 "github.com/openshift-knative/knative-serving-operator/pkg/apis/serving/v1alpha1"
 	"github.com/openshift-knative/knative-serving-operator/pkg/controller/knativeserving/common"
+	"github.com/openshift-knative/knative-serving-operator/pkg/controller/knativeserving/openshift"
```

Comment:
Call me pedantic, but I'd prefer not to see any platform-specific library imported by the controller. I see you're doing it to re-use `checkError`, but since that function is referring to "ServiceMesh" in its message, I don't think that's relevant for all platforms. See below...